ui-backend/catalog-ui-search/pom.xml

```diff
@@ -471,6 +476,7 @@
                         <Export-Package>
                             org.codice.ddf.catalog.ui.metacard.internal,
                             org.codice.ddf.catalog.ui.alias,
+                            org.codice.ddf.catalog.ui.events
```

Comment:
❓ It looks like the types you've defined for that package are in the `catalog-ui-search-api` module, did you mean to add it to the bundle-plugin definition on that pom instead of the `catalog-ui-search` pom?

Reply:
Yes, I did, whoops!